src/transform.ts

```diff
@@ -80,6 +80,17 @@ export interface TimeUnitTransform {
    * The output field to write the timeUnit value.
    */
   as: FieldName;
+
```

Comment:
What about `maxbins` and `interval`?  (the latter should only works for TimeUnit transform, not timeUnit in encoding).

Reply:
`maxbins` added
`interval` seems unnecessary. I can't think of a reason why one would want / need to hide the second output.